plotting/README.md

```diff
@@ -0,0 +1,32 @@
+# Autoreduction Plotting
+
+This document provides documentation for the plotting package as part of the Autoreduction project.
+
+## Ideal Data Structure to Prepare Data into for Visualisation
+
+- Create a dataframe that contains a minimum of Spectrum, X and Y axis columns as seen in the 
+example below 
+    - Note additional columns can be used if needed. 
+
+The spectrum column should act as the id to distinguish between different sections of data to be 
+plotted from one dataframe and can be used to apply pandas group-by functions for more in depth 
+analysis of a given spectrum. 
+
+|Spectrum | X      | Y   | E  |
+| :---    | :---: |:---:| ---:|
+| 1.0     |       |     |     |
```

Comment:
Spectrum should be an integer as it refers to a physical id value . e.g. `1~.0~`